functions/src/content.ts

```diff
@@ -70,7 +70,13 @@ function replaceEmojis(md: string, emojis: Record<string, string>): string {
   for (const k of Object.keys(emojis)) {
     const withColons = `:${k}:`;
     const asEmoji = emojis[k];
-    content = content.replace(new RegExp(withColons, "g"), asEmoji);
+    const contentList = content.split("```");
```

Comment:
@jollypolly123 could you remove these changes from this PR so that it's just the date part?

Reply:
Just changed! Sorry if this isn't proper etiquette, but is that change being reviewed? It's in PR #335

Reply:
I'll take a look